### DISCUSSION TITLE
Build instruction needed

### DISCUSSION THREAD

**Discussion by AndriiZ** [OleksiyPenkov/MyHomeLib#30] [Q&A]
Hi,

Do you have an instruction on how to build MyHomeLib? It would be really helpful for newbies or oldies to make any improvements. 

First of all, IDE version and setup configuration is needed. 

Another question, any extra packages required for a successful build

**Comment by OleksiyPenkov** [comment]
0. Prerequisites: Delphi 11 Alexandria (Free Community Edition). FastMM5, VirtualTreeView (from GetIt Package Manager )
1. Open Program/MHL.groupproj
2. Build and install MHL components 
3. Add Components/MHL Components to the library path
4. Build Myhomelib.exe

**Reply by TimichPL** [reply]
Only GetIt version of VTV will work or this one - https://github.com/JAM-Software/Virtual-TreeView/releases/tag/V8.0.0 - can be used?
